### ISSUE TITLE
SP's are Broken

### ISSUE THREAD

**Issue by cjk12** [convexopt/gpkit#231]
So, the example on the Read the Docs isn't working:

``` python
import gpkit
x = gpkit.Variable('x')
y = gpkit.Variable('y')
gpkit.enable_signomials = True
sp = gpkit.SP(x, [x >= 1-y, y <= 0.1])
sol = sp.localsolve(printing=False, solver=self.solver)
```

I'm getting:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-21-e663144a0e85> in <module>()
      3 y = gpkit.Variable('y')
      4 gpkit.enable_signomials = True
----> 5 sp = gpkit.SP(x, [x >= 1-y, y <= 0.1])
      6 sol = sp.localsolve(printing=False, solver=self.solver)

TypeError: unsupported operand type(s) for -: 'int' and 'Variable'
```

**Comment by whoburg** [comment]
Oops, sorry, this was due to the changes in #228. The docs were updated as part of that, but hadn't been rebuilt on read the docs. That's now been corrected, so that http://gpkit.readthedocs.org/en/latest/advancedcommands.html#signomial-programming shows the correct usage.

In short, the way to enable signomials changed, and is now:

`gpkit.enable_signomials()`.

There is also a corresponding `gpkit.disable_signomials()`, should one want to turn them off after enabling them.

**Comment by whoburg** [comment]
Noticed that advanced commands examples from read the docs didn't all run -- corrected in b1e0a4324d361b23193d16284998efe2b9509887.

@cjk12, if you agree this is fixed, please close.

**Comment by cjk12** [comment]
I have a mosek problem at the moment preventing me from solving, but this appears fixed to me.  Will open a new issue to make sure all examples are unit tests.
